tests/src/Search/HttpClient.php

```diff
@@ -3,14 +3,13 @@
 namespace tests\eLife\Search;
 
 use ComposerLocator;
-use Csa\Bundle\GuzzleBundle\GuzzleHttp\Middleware\MockMiddleware;
+use Csa\GuzzleHttp\Middleware\Cache\MockMiddleware;
 use eLife\ApiClient\HttpClient\Guzzle6HttpClient;
 use eLife\ApiValidator\MessageValidator\JsonMessageValidator;
 use eLife\ApiValidator\SchemaFinder\PathBasedSchemaFinder;
 use GuzzleHttp\Client;
 use GuzzleHttp\HandlerStack;
 use JsonSchema\Validator;
-use test\eLife\ApiSdk\ValidatingStorageAdapter;
```

Comment:
I'd forgotten that this uses a library's test code. :shudders:

Reply:
removing the `use` without removing https://github.com/elifesciences/search/pull/183/files#diff-2a45a15b445df51cb0cb7005d3bf2f6cL30 ?

Reply:
Check the namespace.

Reply:
ah, class "inlined" in the project